Readme.md

```diff
@@ -167,3 +167,37 @@ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 -----------------------------------------------------------------------------
 ```
+
+## Reviewers guide
+
+When reviewing pull request there are common things a reviewer should keep
+in mind.
+
+Reviewing changes to `src/*` and `nuklear.h`:
+
+* Ensure C89 compatibility.
+* The code should work for several backends to an acceptable degree.
+* Recommend simple optimizations.
+  * Pass small structs by value instead of by pointer.
+  * Use local buffers over heap allocation when possible.
+* Check that the coding style is consistent with code around it.
+  * Variable/function name casing.
+  * Indentation.
+  * Curly bracket (`{}`) placement.
+* Ensure that the contributer have bumped the appropriate version in
+  [package.json](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/package.json)
+  and added their changes to the
+  [CHANGELOG](https://github.com/Immediate-Mode-UI/Nuklear/blob/master/src/CHANGELOG).
+* Have at least one other person review the changes before merging.
+
+Reviewing changes to `demo/*`, `example/*` and other files in the repo:
+
+* Focus on getting working code merged.
+  * We want to make it easy for people to get started with Nuklear, and any
+    `demo` and `example` improvements helps in this regard.
+* Use of newer C features is not discouraged.
```

Comment:
Just a nit - backends etc. can be written in any language, not just C :wink:. Otherwise you can merge this PR. Thanks for putting your thoughts in improving the merge process!